Framework/CMakeLists.txt

```diff
@@ -95,6 +95,7 @@ target_link_libraries(QualityControl
                              ms_gsl::ms_gsl
                              QualityControlTypes
                              O2::Mergers
+                             O2::DataSampling
```

Comment:
@Barthelemy Is it possible to link the library, but do not complain if it is not found?

Reply:
yes, with generator expressions: https://cmake.org/cmake/help/v3.2/manual/cmake-generator-expressions.7.html

Reply:
Probably something like `"$<$<O2::DataSampling>:O2::DataSampling>"` (I don't remember exactly)

Reply:
Thanks, probably `$<$<TARGET_EXISTS:O2::DataSampling>:O2::DataSampling>` should work. I am giving it a try now